config/default_configs/default_config.yml

```diff
@@ -145,6 +145,15 @@ vert_diff:
 hyperdiff:
   help: "Hyperdiffusion [`ClimaHyperdiffusion` (or `true`; default), `none` (or `false`)]"
   value: "CAM_SE"
+smagorinsky_lilly:
+  help: "Smagorinsky-Lilly diffusive closure [`false` (default), `true`]"
+  value: false
+c_smag: 
```

Comment:
`c_smag` is already a ClimaParameter, so let's not duplicate it here. This will also cause some confusion for users as there are two places to set `c_smag` now.